### DISCUSSION TITLE
Star History

### DISCUSSION THREAD

**Discussion by johnblommers** [vkbo/novelWriter#748] [General]
Did you know that somebody wrote a [Star History](http://www.gitstarhistory.com/#vkbo/novelWriter) plotter?

It seems that novelWriter got a real boost early this year:

![Star History](https://user-images.githubusercontent.com/8484427/114959130-4988b880-9e19-11eb-9ce5-49dd014590f7.png)

**Comment by vkbo** [comment]
Hah, nice. Yeah, it got quite busy there for a while!

**Reply by johnblommers** [reply]
Congratulations. The recognition is well-deserved.

**Comment by longqzh** [comment]
I checked the date, and guess the sudden rising is caused by hacker news :)
